test/components/views/elements/MemberEventListSummary-test.js

```diff
@@ -0,0 +1,556 @@
+const expect = require('expect');
+const React = require('react');
+const ReactDOM = require("react-dom");
+const ReactTestUtils = require('react-addons-test-utils');
+const sdk = require('matrix-react-sdk');
+const MemberEventListSummary = sdk.getComponent('views.elements.MemberEventListSummary');
+
+const testUtils = require('../../../test-utils');
+describe.only('MemberEventListSummary', function() {
+    let sandbox;
+    let parentDiv;
+
+    const generateTiles = (events) => {
+        return events.map((e) => {
+            return (
+                <div key={e.getId()} className="event_tile">
+                    Expanded membership
+                </div>
+            );
+        });
+    };
+
+    const generateMembershipEvent = (eventId, parameters) => {
```

Comment:
I'd have to modify it to include the ability to set previous content, though mocking out `MatrixEvent` seems wrong. I'll just use a real `MatrixEvent` and override `target`.